src/internal/clusterstate/v2.6.0/pfsdb.go

```diff
@@ -578,6 +584,7 @@ func branchlessCommitsPFS(ctx context.Context, tx *pachsql.Tx) error {
 		}
 	}
 	// map <project>/<repo>@<branch>=<id> -> <project>/<repo>@<id>; basically replace '@[-a-zA-Z0-9_]+)=' -> '@'
+	log.Info(ctx, "removing branch from internal keys (this can be an expensive operation)")
```

Comment:
Can we have some kind of done line for this.  Maybe using the Span functionality?